Functions/Coverage.Tests.ps1

```diff
@@ -282,21 +285,21 @@ InModuleScope Pester {
             Enter-CoverageAnalysis -CodeCoverage "$root\TestScriptWithConfiguration.ps1" -PesterState $testState
 
             It 'Has the proper number of breakpoints defined' {
-                $testState.CommandCoverage.Count | Should Be 7
+                $testState.CommandCoverage.Count | Should Be 8
```

Comment:
Judging from comments in the scripts, this should stay `7`: new resource parameter `# does NOT trigger breakpoint`